artifacts/oopsla25-bv-decide/Dockerfile

```
@@ -1,55 +1,124 @@
 FROM ubuntu:24.04
-WORKDIR /code
+
+# Set up non-interactive mode for apt-get
+ENV DEBIAN_FRONTEND=noninteractive
+ENV TZ=Etc/UTC
 
 # install dependencies
-RUN apt-get update -y \
-    && DEBIAN_FRONTEND="noninteractive" apt-get install --yes \
+RUN apt-get update && \
+    apt-get install --yes \
```

```suggestion
RUN apt-get update -y && \
    apt-get install --yes \
```
Was the loss of the `-y` flag intentional?

---

Yes!